ui/components/app/currency-input/currency-input.js

```diff
@@ -230,7 +246,8 @@ CurrencyInput.propTypes = {
   asset: PropTypes.shape({
     address: PropTypes.string,
     symbol: PropTypes.string,
-    decimals: PropTypes.number,
+    decimals: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
```

Comment:
What causes this to be a string? Can the value be normalized to a number on selection, before it's passed into this component?

Reply:
There's a mistype somewhere. The commit (a5a5417a23a2647b52064a22ff178bd563687f96) was made in response to a console error (not thrown) regarding an invalid prop type being passed. Also doesn't hurt to have a bit of extra robustness anyways.